presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -2556,9 +2556,9 @@ public void testJoinWithNullValues()
     @Test
     public void testJoinWithMultipleInSubqueryClauses()
     {
-        QueryTemplate.Parameter type = new QueryTemplate.Parameter("type", "");
-        QueryTemplate.Parameter condition = new QueryTemplate.Parameter("condition", "true");
-        QueryTemplate queryTemplate = new QueryTemplate(
+        QueryTemplate.Parameter type = parameter("type").of("");
```

Comment:
When you use `parameter` outside of `QueryTemplate.of` method then it looks a little bit akwaryd. But renaming this to `queryParameter` will make the other case look bad.